EDDI/MainWindow.xaml.cs

```diff
@@ -1008,27 +1009,42 @@ public static bool IsAdministrator()
 
         private void ttsVoiceDropDownUpdated(object sender, SelectionChangedEventArgs e)
         {
-            ttsUpdated();
+            if (sender is ComboBox comboBox && comboBox.IsLoaded)
```

Comment:
I'm curious why we are testing the sender now?

Reply:
Can we instead cast to `FrameworkElement` in all of these? That would make the event handler robust to a designer changing the control type in the future.

Reply:
Done.